src/screens/Profile/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
```

Comment:
Remove this file and move it's logic to profile.tsx, also rename Profile folder to lowercase